cmd/flags.go

```diff
@@ -40,10 +40,18 @@ func newStringOpts() DocOpt[string] {
 	return DocOpt[string]{}
 }
 
+func newStringSliceOpts() DocOpt[[]string] {
+	return DocOpt[[]string]{}
+}
+
 func stringFlag(flags *pflag.FlagSet, name, usage string, opts ...DocOpt[string]) {
 	addFlag(name, usage, opts, flags.String, flags.StringP)
 }
 
+func stringSliceFlag(flags *pflag.FlagSet, name, usage string, opts ...DocOpt[[]string]) {
```

Comment:
Is there a reason `name` is untyped? All the usages are strings, is there a situation where this wouldn't be expected?

Reply:
`name, usage string` is just shorthand for `name string, usage string`.